install/infra/single-cluster/aws/README.md

```diff
@@ -192,6 +192,18 @@ documentaion](https://www.gitpod.io/docs/self-hosted/latest/getting-started#step
 
 ## Troubleshooting
 
+### KOTS pods fail to deploy
+
+Sometimes, the pods deployed when executing `kubectl kots install gitpod` fail to deploy due to issues with mounting their disks:
+
+```lucas@Lucass-MBP aws % kubectl get pods -A
+NAMESPACE      NAME                                       READY   STATUS              RESTARTS   AGE
+gitpod         kotsadm-minio-0                            0/1     ContainerCreating   0          2m28s
+gitpod         kotsadm-postgres-0                         0/1     Init:0/2            0          2m28s
+```
+
+This can happen when the wrong `image_id`  was used in the `.tfvars` file. The id needs to respect both the region as well as the Kubernetes version and can be found [here](https://cloud-images.ubuntu.com/docs/aws/eks/). 
```

Comment:
```suggestion
This can happen when the wrong `image_id`  was used in the `.tfvars` file. The ID needs to respect both the region as well as the Kubernetes version and can be found [here](https://cloud-images.ubuntu.com/docs/aws/eks/). 
```